plenum/test/cli/helper.py

```diff
@@ -611,32 +611,32 @@ def getOldIdentifiersForActiveWallet(cli):
 def createAndAssertNewCreation(do, cli, keyringName):
     oldIdentifiers = getOldIdentifiersForActiveWallet(cli)
     do('new key', within=2,
-       expect=["Key created in keyring {}".format(keyringName)])
+       expect=["Key created in wallet {}".format(keyringName)])
     assert len(cli._activeWallet.identifiers) == oldIdentifiers + 1
 
 
 def createAndAssertNewKeyringCreation(do, name, expectedMsgs=None):
     finalExpectedMsgs = expectedMsgs if expectedMsgs else [
-           'Active keyring set to "{}"'.format(name),
-           'New keyring {} created'.format(name)
+           'Active wallet set to "{}"'.format(name),
+           'New wallet {} created'.format(name)
         ]
-    do('new keyring {}'.format(name), expect=finalExpectedMsgs)
+    do('new wallet {}'.format(name), expect=finalExpectedMsgs)
 
 
 def useAndAssertKeyring(do, name, expectedName=None, expectedMsgs=None):
     keyringName = expectedName or name
     finalExpectedMsgs = expectedMsgs or \
-                        ['Active keyring set to "{}"'.format(keyringName)]
-    do('use keyring {}'.format(name),
+                        ['Active wallet set to "{}"'.format(keyringName)]
+    do('use wallet {}'.format(name),
        expect=finalExpectedMsgs
     )
 
 
 def saveAndAssertKeyring(do, name, expectedName=None, expectedMsgs=None):
```

Comment:
Should `saveAndAssertKeyring` be renamed to `saveAndAssertWallet`?
Is such a change in the scope?